src/azure-cli/azure/cli/command_modules/synapse/manual/commands.py

```diff
@@ -207,6 +207,11 @@ def get_custom_sdk(custom_module, client_factory):
         client_factory=cf_kusto_script,
     )
 
+    synapse_link_connection_sdk = CliCommandType(
+        operations_tmpl='azure.synapse.artifacts.operations#linkconnectionOperations.{}',
+        client_factory=None,
```

Comment:
```suggestion
        client_factory=cf_synapse_link_connection,
```
You can define the client factory here